tsl/test/sql/decompress_vector_qual.sql

```diff
@@ -73,6 +73,19 @@ select count(*) from vectorqual where metric4 is null;
 select count(*) from vectorqual where metric4 is not null;
 
 
+-- Vectorized filters also work if we have only stable functions on the right
+-- side that can be evaluated to a constant at run time.
+set timescaledb.debug_require_vector_qual to 'only';
+select count(*) from vectorqual where ts > '2021-01-01 00:00:00'::timestamptz::timestamp;
+select count(*) from vectorqual where ts > '2021-01-01 00:00:00'::timestamp - interval '1 day';
+-- Note that here the comparison itself is a stable, not immutable, operator,
+-- which we don't vectorize yet.
+set timescaledb.debug_require_vector_qual to 'forbid';
+select count(*) from vectorqual where ts > '2021-01-01 00:00:00'::timestamptz;
```

Comment:
Could you say more about why we cannot vectorize this? (Just a matter of recognizing thesetype of expressions? )

Reply:
Maybe we could rewrite it to 'timestamp > timestamp' with a different constant, but it needs more thought.

Reply:
Thank you for the clarification.